packages/stream/get-stream/get-stream.test.js

```diff
@@ -116,14 +124,35 @@ describe('Get Stream Module', function () {
       expect(result).to.have.property('status').to.be.an('object')
       expect(result).to.have.property('data').to.be.an('object')
       expect(result.data).to.have.property('id').to.be.equal(streamId)
-      expect(result.data).to.have.property('name').to.be.a('string')
+      expect(result.data).to.have.property('name').to.be.a('string').to.not.be
+        .empty
       expect(result.data).to.have.property('slug').to.be.a('string')
+      expect(result.data).to.have.property('description').to.be.a('string')
+      expect(result.data).to.have.property('start_time').to.be.a('string').to
+        .not.be.empty
+      expect(result.data).to.have.property('end_time').to.be.a('string').to.not
+        .be.empty
+      expect(result.data).to.have.property('billing_start').to.be.a('string').to
+        .not.be.empty
+      expect(result.data).to.have.property('billing_end').to.be.a('string').to
+        .not.be.empty
+      expect(result.data).to.have.property('billing_end').to.be.a('string').to
+        .not.be.empty
```

Comment:
@ingrid011 I think this is a duplicate of the one above it.

Reply:
@faiq-naufal ops, sorry mas, done at commit [e1556e6](https://github.com/inlivedev/inlive-js-sdk/pull/11/commits/e1556e62bb157099cc9e1747c590138465400b82). Thank you.